### PR TITLE
meta: update issue template

### PR DESCRIPTION
**Pull request by berbiche**
### Description

Tested on a temporary local repo (issues cannot be filled on forks).

With this PR when a new issue is created, me and @rycee will be automatically tagged.
This can be modified to add more people (i.e. if there's an HM team in the nix-community org we can tag `@nix-community/home-manager`).

The PR template can also be updated to have a similar template.

**What the user sees**:
image

**Rendered**:
image

References:
- https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
- https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
- https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema

This new format is currently in "beta" so we can alternatively update the format to the following:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#markdown-issue-template

### Checklist

- [ ] Change is backwards compatible.

- [ ] Code formatted with `./format`.

- [ ] Code tested through `nix-shell --pure tests -A run.all`.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/doc/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

  - [ ] Added myself and the module files to `.github/CODEOWNERS`.

### CODE REVIEW

**Comment by sumnerevans** [comment]
Can you still make a free-form issue for things like feature requests?

**Comment by berbiche** [comment]
> Can you still make a free-form issue for things like feature requests?

We can add a template for that

**Comment by berbiche** [comment]
@sumnerevans done:

image

image

**Comment by berbiche** [comment]
You can test it here: https://github.com/berbiche/temp-home-manager

I will delete the repo once this is merged/closed.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Ask for a new feature to be added (module, program, etc.)
+title: '<title>'
```
Comment (sumnerevans) [inline .github/ISSUE_TEMPLATE/feature-request.md:4]: I don't love how this auto-populates the field:

image

Comment (berbiche) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Fixed

**Review by sumnerevans — APPROVED** [review]

**Comment by rycee** [comment]
Thanks, very nice! Merged to master now.

**Pull request merged**